common/tools/turborepo-remote-cache/README.md

```diff
@@ -0,0 +1,149 @@
+```
+
+For development, you can use the following command:
+
+```bash
+docker build -t azure-turborepo-remote-cache --build-arg NODE_ENV=development .
+```
+
+In order to run the cache in Docker, you will need to log into Azure to create the token on your local system. You can do this using the Azure CLI:
+
+```bash
+az login
+```
+
+Using an `.env` file is supported so you can run the container with the following command where you mount the local `.azure` directory to the container's `/root/.azure` directory for local development:
+
+```bash
+docker run -p 3000:3000 -it -v ~/.azure:/root/.azure azure-turborepo-remote-cache
+```
+
+When running in production, you can use the following command leaving out the `-v` flag and set the `NODE_ENV` variable to `production`:
+
+```bash
+docker run -p 3000:3000 -it azure-turborepo-remote-cache -e NODE_ENV=production
+```
+
+Running in production mode is recommended for production use. You can do this by setting the `NODE_ENV` environment variable to `production`:
+
+```bash
+docker run -p 3000:3000 \
+  -it \
+  -e NODE_ENV=production \
+  -e AZURE_CACHE_PORT=3000 \
+  -e PACKAGE_VERSION=1.0.0 \
+  -e CACHE_LOG_LEVEL=info \
+  -e TURBO_TOKEN=token1,token2 \
+  -e AZURE_STORAGE_ACCOUNT=your_account_name \
+  -e AZURE_STORAGE_CONTAINER_NAME=your_container_name \
+  azure-turborepo-remote-cache
+```
+
+### Configuring TurboRepo
+
+To configure TurboRepo Build to use the remote cache, you need to create the `.turbo/config.json` file in the root of your project and add the following configuration, where the team token is one of the tokens you set in the `TURBO_TOKEN` environment variable:
+
+```json
+{
+  "apiurl": "<your_api_url>",
+  "teamslug": "<your_team_slug>",
+  "token": "<your_token>"
+}
+```
+
+### Creating a New Azure Storage Account
+
+If you don't have an Azure Storage account, you can create one using the Azure CLI. First, log in to your Azure account:
+
+```bash
+az login
+```
+
+Then, create a new resource group:
+
+```bash
+az group create --name myResourceGroup --location eastus
+```
+
+Next, create a new storage account with our bicep template:
+
+```bash
+az deployment group create \
+  --resource-group <RESOURCE_GROUP_NAME> \
+  --template-file cache.bicep \
+  --parameters storageAccountName=mycustomstorage containerName=mycustomcontainer
+```
+
+## Troubleshooting
+
+### Logging
+
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `CACHE_LOG_LEVEL` environment variable to `info`.
+
+## Contributing
+
+If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md) to learn more about how to build and test the code.
+
+This module's tests are a mixture of live and unit tests, which require you to have an Azure Notification Hubs instance. To execute the tests you'll need to run:
```

Comment:
notification hubs?

Reply:
Ah must be left over from copy and paste. will remove.